client/core/core.go

```diff
@@ -2745,7 +2766,9 @@ func handleTradeSuspensionMsg(c *Core, dc *dexConnection, msg *msgjson.Message)
 		// Clear the bookie associated with the suspended market.
 		dc.booksMtx.Lock()
 		if bookie := dc.books[sp.MarketID]; bookie != nil {
-			dc.books[sp.MarketID] = newBookie(func() { c.unsub(dc, sp.MarketID) })
+			// Old bookie and it's feeds get garbage collected. Any orderbook
+			// subscriptions remain, and are handled by the new bookie.
```

Comment:
Hmmm, I was considering closed feeds channels found by the marketSyncer loop to indicate a bug, but one that won't cause a loop to spin and eat cpu, just log.  My thinking was that the proper use is for the consumer to `cl.feedLoop.Stop()` to break the loop, but with this suspend handling I'm reevaluating that.

In fact, I don't think I follow the reasoning for making a new bookie like this on suspend.  The feed subscribers are left high and dry.... Seems like you are correct that the right thing to do is to `delete(dc.books, sp.MarketID)` and loop through the feeds closing all their channels, thus stopping the `marketSyncer`s created by rpc/webserver so that they can resubscribe at a future point (trade resumption).

Reply:
> In fact, I don't think I follow the reasoning for making a new bookie like this on suspend.

Now that you mention it, I don't either. Hmm.

Reply:
Crap there are several missing pieces in suspend resume to make this work correctly.  Hopefully we can move forward with that next.  For now I noted in https://github.com/decred/dcrdex/pull/576/commits/190584776c05784a41ba1c4e02dda67fec459f55 that there is an issue with suspend and what needs to happen.